tests/virt_autotest/reboot_and_wait_up_upgrade.pm

```diff
@@ -18,7 +18,7 @@ use base "reboot_and_wait_up";
 
 sub run {
     my $self    = shift;
-    my $timeout = 1200;
+    my $timeout = 600;
```

Comment:
No prove to decrease it? I know there is a bug during this step which die soon.
If the step of upgrade succeed, I suspect that if the timeout is enough.

Reply:
The worker tripples the timeout value. Generally upgrade takes less than 30min.

Reply:
i see, ipmi has default time scale 3 :)